src/main/java/org/symphonyoss/symphony/messageml/util/ShortID.java

```diff
@@ -0,0 +1,128 @@
+package org.symphonyoss.symphony.messageml.util;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+/**
+ * Short id generator. Url-friendly. Non-predictable. Cluster-compatible.
+ *
+ * <p></p>
+ * Inspired from <a href="https://github.com/dylang/shortid">shortid</a>
+ */
+public class ShortID {
+
+  public static final String DEFAULT_ALPHABET =
+      "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_-";
+
+  public static final String DOLLARAT_ALPHABET =
+      "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ$@";
```

Comment:
Do we want to generate css IDs with `_`, `-`, `$` and `@`?

Reply:
😄 removed